tests/subsys/rtio/rtio_api/src/main.c

```diff
@@ -414,18 +414,6 @@ void test_rtio_multiple_chains(void)
 	test_rtio_multiple_chains_(&r_multi_con);
 }
 
-void test_main(void)
-{
-	TC_PRINT("imxrt1010 RTIO\n");
-	ztest_test_suite(rtio_spsc_test,
-			 ztest_1cpu_unit_test(test_produce_consume_size1),
```

Comment:
Looks like these were using ztest_1cpu before.

They should be in their own suite using: [ztest_simple_1cpu_before/after](https://github.com/zephyrproject-rtos/zephyr/blob/66cfe4f430dc960432fa4354e6ca383486f83f58/subsys/testsuite/ztest/include/zephyr/ztest_test_new.h#L345)

Reply:
They were, but not really necessarily needed here

Reply:
Understood! Thanks.